crates/solver/src/orderbook.rs

```diff
@@ -20,7 +20,12 @@ impl OrderBookApi {
 
     pub async fn get_auction(&self) -> Result<AuctionWithId> {
         let url = self.base.join("api/v1/auction")?;
-        let auction = self.client.get(url).send().await?.json().await?;
+        let response = self.client.get(url).send().await?;
+        if let Err(err) = response.error_for_status_ref() {
+            let body = response.text().await;
+            return Err(anyhow::Error::new(err).context(format!("body: {:?}", body)));
```

Comment:
I guess we should also make sure that the status code is included in the message. Not sure if that would be the case right now.

Reply:
I'm pretty sure it is included because the Debug/Display impl of the reqwest error includes it. (But not 100% sure)

Reply:
Its included in the `ErrorForStatus` error.